src/Arcus.Observability.Telemetry.Core/Extensions/ILoggerExtensions.cs

```diff
@@ -31,6 +31,16 @@ public static class ILoggerExtensions
             + ContextProperties.DependencyTracking.IsSuccessful + "} - Context: {@"
             + ContextProperties.EventTracking.EventContext + "})";
 
+        private const string ServiceBusDependencyFormat =
+            MessagePrefixes.Dependency + " Azure Service Bus {"
```

Comment:
`Azure Service Bus` is the dependency type to use

Reply:
Changed that.